docs/features/diff-viewer/setup.md

```diff
@@ -1,96 +1,16 @@
-#Diff Viewer Installation and Configuration Guide
+#Configuration Guide
 
-This guide describes how you can set up the Diff Viewer feature.
+This guide describes how you can configure the Diff Viewer feature.
 The Diff Viewer feature makes it possible to compare two different builds against each other.
-In three easy step, you are ready to compare your own builds:
+In two easy step, you are ready to compare your own builds:
 
-1. [Install  GraphicsMagick](#1-install-graphicsmagick)
-2. [Add a comparison configuration](#2-add-a-comparison-configuration)
-3. [Start comparison](#3-start-comparison)
-4. [Delete a comparison](#4-delete-a-comparison)
+1. [Add a comparison configuration](#2-add-a-comparison-configuration)
+2. [Start comparison](#3-start-comparison)
+3. [Delete a comparison](#4-delete-a-comparison)
```

Comment:
bitte dritten Schritt entfernen, der gehört wohl irgendwie nicht zu den "In two easy step, you are ready to compare your own builds"